safe/gui/tools/wizard/step_kw57_extra_keywords.py

```diff
@@ -0,0 +1,202 @@
+# coding=utf-8
+"""InaSAFE Wizard Step Extra Keywords."""
+
+from PyQt4.QtGui import (
+    QLineEdit, QDateTimeEdit, QDoubleSpinBox, QComboBox, QCheckBox
+)
+from PyQt4.QtCore import Qt
+from collections import OrderedDict
+
+from datetime import datetime
+import pytz
```

Comment:
import are coming before, especially when it's python builtins. (according to PEP). PyCharm will update it, just for reference

Reply:
Fixed.